### PR TITLE
blade template promotions and types

### PR DESCRIPTION
**Pull request by NTBNgoc**
blade template promotions and types

### CODE REVIEW

File: resources/views/promotions/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                </div>
+                <div class="clearfix"></div>
+            </div>
+            <div class="x_content">
+                @if( $promotions->count() > 0 )
+                    <table class="table table-striped" id="datatable">
+                        <thead>
+                            <tr>
+                                <th>STT</th>
+                                <th>Dicount</th>
```
Comment (BunnyPi04) [inline resources/views/promotions/index.blade.php]: i18n

File: resources/lang/en/admin.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?php 
+
+    return [
+    	'YumYumStore' => 'YumYum Store',
+        'store' => 'Stores',
```
Comment (BunnyPi04) [inline resources/lang/en/admin.php]: tab

File: resources/views/promotions/index.blade.php (hunk 1 above)
Comment (NTBNgoc) [reply resources/views/promotions/index.blade.php]: em sửa rồi ạ

File: resources/views/types/edit.blade.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                    <div class="alert alert-success alert-dismissible fade in">
+                        <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">×</span></button>
+                        {{ session('status') }}
+                    </div>
+                    @endif
+                    {!! Form::open(['method' => 'POST', 'route' => ['type.getEdit', $data->id], 'enctype' => 'multipart/form-data']) !!}
+                        <div class="form-group {{ $errors->has('name') ? 'has-error' : '' }}">
+                            {!! Form::label(trans('messeage.typeName')) !!}
+                            {!! Form::text('name', $data->name, ['class' => 'form-control', 'placeholder' => trans('messeage.typeName')]) !!}
+                            @if($errors->has('name'))
```
Comment (BunnyPi04) [inline resources/views/types/edit.blade.php]: convention

**Pull request merged**